docs/bokeh/source/docs/releases/3.7.0.rst

```diff
@@ -11,3 +11,11 @@ Bokeh version ``3.7.0`` (January 2025) is a minor milestone of Bokeh project.
 * Added support for dual canvas and DOM rendering to ``Legend`` annotation (:bokeh-pull:`14028`)
 * Added support for custom marker definitions to ``Scatter`` glyph (:bokeh-pull:`14165`)
 * Added support for stateful action tools, especially ``CustomAction`` (:bokeh-pull:`14143`)
+
+API and breaking changes
+------------------------
+
+BokehJS
+^^^^^^^
+
+* Changed the type of ``TemplateView.action_views`` from ``ViewStorage<Action>``to ``ViewOf<Action>[]``.
```

Comment:
As requested by @hoxbro. I'm not sure if this is the right place for this, but we don't have anything else established for communicating API and breaking changes. Maybe except for a blog posts, but we don't track that these in the repository. I recall @bryevdv previously was against listing changes like theses in release notes. @bokeh/dev, if you have any thoughts on this.

Reply:
I thought we put changes like this on the migration guide wiki page so that things can be updated much more easily without requiring full documentation site builds and uploads.

Reply:
I completely forgot about that wiki [page](https://github.com/bokeh/bokeh/wiki/Migration-Guides). It's telling that it hasn't been updated since bokeh 3.2. I think we need to collect such changes in PRs and perhaps then copy/move them to the wiki (as the most up-to-date source of information).